modules/math/src/geometry/types.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Type for Bounding Box computing
+ */
+export type Geometry = {
```

Comment:
`Geometry` is the `{attributes, indices?, mode}` thing. This is a bounding box
```suggestion
export type BoundingBox = {
```